media.css

```diff
@@ -0,0 +1,125 @@
+@media (max-width: 1550px) {
+  /*.other {
+    display: flex;
+    flex-direction: column;
+  }*/
+
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)